share/completions/lxc.fish

```diff
@@ -0,0 +1,51 @@
+function __fish_lxc_no_subcommand -d 'Test if lxc has yet to be given the command'
+    for i in (commandline --tokenize --cut-at-cursor --current-process)
+        if contains -- $i config copy delete exec file help image launch list move network profile publish remote restore restart snapshot start stop
+            return 1
+        end
+    end
+    return 0
+end
+
+function __fish_lxc_list_containers
+    lxc list -c n | grep '| \w' | sed -e 's/\(|\| \)*//g'
+end
+
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments config --description 'Manage configuration.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments copy --description 'Copy containers within or in between lxd instances.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments delete --description 'Delete containers or container snapshots.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments exec --description 'Execute the specified command in a container.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments file --description 'Manage files on a container.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments finger --description 'Fingers the LXD instance to check if it is up and working.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments help --description 'Usage: lxc [subcommand] [options]'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments image --description 'Manipulate container images.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments info --description 'List information on LXD servers and containers.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments init --description 'Initialize a container from a particular image.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments launch --description 'Launch a container from a particular image.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments list --description 'Lists the available resources.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments manpage --description 'Prints all the subcommands help.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments monitor --description 'Monitor activity on the LXD server.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments move --description 'Move containers within or in between lxd instances.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments network --description 'Manage networks.'
+complete --condition '__fish_lxc_no_subcommand' --command lxc --no-files --arguments pauze --description 'Changes state of one or more containers to pause.'
```

Comment:
Typo: I assume it's "pau**s**e"? Also "changes state to X" sounds a bit weird - why not just "pauses container" and similar for the other states.